### ISSUE TITLE
media indicator: there should be a space between offscreen text and duration

### ISSUE THREAD

**Issue by andrew-nowak** [BBC-archive/psammead#698]
**Describe the bug**
![image](https://user-images.githubusercontent.com/10963046/60085283-eb18b480-9730-11e9-8168-1cbe3427d51c.png)

**To Reproduce**
Steps to reproduce the behaviour:

1. Open a front page with media indicators
2. Disable CSS
3. Scroll to media promo

**Expected behaviour**
There should be a space between the offscreen text and the duration string

**Solutions**
1. Insert a space
2. Drop offscreen text here; it was only added to support user agents which do not respect `aria-hidden: true` which is not a case handled anywhere else in simorgh/psammead. See also bbc/simorgh#2096

**Comment by dr3** [comment]
Removing v1 label due to https://github.com/bbc/simorgh/pull/2172

**Comment by andrew-nowak** [comment]
There's no offscreen text in the media indicator since bbc/simorgh#2172
